tilequeue/query/postgres.py

```diff
@@ -67,9 +64,15 @@ def __init__(self, sources, query_generator):
     def __call__(self, zoom, bounds):
         queries = []
         for source in self.sources:
-            if source.start_zoom <= zoom:
-                query = self.query_generator(source.template, bounds, zoom)
-                queries.append(query)
+            template_queries = []
+            for template_spec in source.template_specs:
+                if template_spec.zoom_start <= zoom < template_spec.zoom_stop:
```

Comment:
Exclusive. I put in a comment in the [queries.yaml in the vector-datasource pr](https://github.com/tilezen/vector-datasource/pull/1365/files#diff-fc49e5c256610bc0b32a7de5a0520835R20), but that doesn't help reading the code. I added in comments in 0e9e0db.

Would you prefer that it just be renamed to something more verbose but clear like `zoom_stop_exclusive`? It's nice that start/stop feel symmetric, but having the extra characters for clarity seems worthwhile, and don't bother me :)